bot/exts/utils/bot_info.py

```diff
@@ -45,7 +45,10 @@ async def stats(self, inter: ApplicationCommandInteraction) -> None:
         embed.add_field(name="Servers", value=len(guilds))
         embed.add_field(
             name="Members",
-            value=sum([guild.member_count for guild in guilds]),
+            value=len(
+                set().union(set(user.id for user in guild.members) for guild in guilds)
+            )  # Length of the union of every single UID
+            - 1,  # Remove the bot itself
         )
```

Comment:
Maybe we could add this as a new field? `Unique members`?

I feel like it's nice to have the "normal" member count too, since that's what most bots are probably using for counting their members. Otherwise this could be unfair for comparisons.